PYME/Acquire/Hardware/Piezos/piezo_e816_dll.py

```diff
@@ -294,6 +295,13 @@ def _Loop(self):
                     # logging.debug('Moving piezo to target: %f' % (pos[0],))
 
                 if np.allclose(self.position, self.targetPosition, atol=self._target_tol):
+                    if not self.onTarget and self._spool_controller:
+                        try:
+                            self._spool_controller.spooler.evtLogger.logEvent('PiezoOnTarget',
```

Comment:
the correct way to log an event is:

```python
from PYME.Acquire import eventLog
eventLog.logEvent(....)
```

This automatically logs the event with anything which has registered as an event consumer (ie. the spool controller will get the event - you don't need to hardcode the routing.)

Reply:
Also, you should generally **not** supply a timestamp - leave that for the spooler. This is mostly to make sure that the simulator works (it runs faster than real time and spoofs it's timestamps), but also allows us to, e.g. use a hardware timestamp source in the future.

Reply:
> I've updated eventLog.py to add a bit more info about how the event logging is supposed to work.

Thanks for this, @David-Baddeley  - sorry I'd missed that.

Reply:
For the timestamp, is it preferable to supply it here since we are in a thread (and this piezo will never be used by the simulator)?